### PR TITLE
Health logs for find / replace in files, instant search usage.

### PR DESCRIPTION
**Pull request by abose**
Tracks
Instant,new,regexp,casesensitive , search, replace
also next,prev,last,first in search results.

### CODE REVIEW

File: src/utils/HealthLogger.js [hunk 1]
```diff
@@ -174,6 +174,22 @@ define(function (require, exports, module) {
         setHealthDataLog("ProjectDetails", FIFLog);
     }
 
+    /**
+     * Increments health log count for a particular kind of search done
+     * @param {string} searchType The kind of search type that needs to be logged- should be a js var compatible string
+     */
+    function searchDone(searchType) {
+        var searchDetails = getHealthDataLog("searchDetils");
```
Comment (MarcelGerber) [inline src/utils/HealthLogger.js]: typo: searchDetils

File: src/utils/HealthLogger.js [hunk 2]
```diff
@@ -181,7 +197,22 @@ define(function (require, exports, module) {
...
+    exports.SEARCH_INSTANT            = "searchInstant";
+    exports.SEARCH_ON_RETURN_KEY      = "searchOnReturnKey";
+    exports.SEARCH_REPLACE_ALL        = "searchReplaceAll";
+    exports.SEARCH_NEXT_PAGE          = "searchNextPage";
+    exports.SEARCH_PREV_PAGE          = "searchPrevPage";
+    exports.SEARCH_LAST_PAGE          = "searchLastPage";
+    exports.SEARCH_FIRST_PAGE         = "searchFirstPage";
+    exports.SEARCH_REGEXP             = "searchRegExp";
+    exports.SEARCH_CASE_SENSITIVE     = "searchCaseSensitive";
+    // A new search context on serach bar up-Gives an idea of number of times user did a discrete search
```
Comment (MarcelGerber) [inline src/utils/HealthLogger.js]: typo: serach

**Comment by nethip** [comment]
@abose can you take of the review comments pointed out by @MarcelGerber ?

**Comment by abose** [comment]
@nethip fixed. I think this is good to be merged now.

**Pull request merged**